lib/shopping_cart.rb

```diff
@@ -0,0 +1,64 @@
+class ShoppingCart
+  attr_reader :name,
+              :capacity,
+              :products,
+              :sorted,
+              :breakdown
```

Comment:
you do not need to have an attr_reader for every attribute you have.. only the ones you wish to call as methods.